media/js/helpfulness.js

```diff
@@ -0,0 +1,80 @@
+(function(win, doc, $) {
+    // no localStorage, no helpfulness rating
+    if (('localStorage' in win)) {
+        var stfu = localStorage.getItem('helpful-stfu') === 'true'; // true if ever clicked stfu
+        var asked_recently = parseInt(localStorage.getItem(document.location + '#answered-helpful')) > Date.now();
```

Comment:
You can use `doc` instead of document and for JS validation purposes, you should add `10` as the second argument to `parseInt`